template.rb

```diff
@@ -60,6 +60,12 @@ def apply_template!(template_root)
   apply '.template/addons/docker/template.rb'
   apply '.template/addons/semaphore/template.rb'
 
+  if ENV['CI']
+    apply '.template/addons/heroku/template.rb'
+  elsif yes?("Would you like to add heroku?")
```

Comment:
How about add an action to guide user?

something like this

```ruby
Would you like to add Heroku? (yes/no)
```

Reply:
@gutakk yeah that I have also in my mind. We will make it an one line once reviewing done and ready for merging ✌️